src/pages/SearchPage/ui/SearchPage.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component } from 'react';
+import { ItemCard } from '@entities/ItemCard';
+import { Item } from '@shared/api/items';
+import { Loader } from '@shared/ui/Loader/Loader';
+import css from './style.module.css';
+import '@shared/styles/global.css';
+
+interface Props {
+  items: Item[];
+  isFetching: boolean;
+  isResultEmpty: boolean;
+}
+export class SearchPage extends Component<Props> {
+  renderItems() {
+    return this.props.items.map((item: Item) => (
+      <ItemCard
+        key={item.mal_id}
+        mal_id={item.mal_id}
+        title={item.title}
+        year={item.year}
+        duration={item.duration}
+        episodes={item.episodes}
+        favorites={item.favorites}
+        images={item.images}
+      />
+    ));
+  }
+
+  render() {
+    const { isFetching, isResultEmpty } = this.props;
+    return (
+      <div className="container">
+        <div className={css.wrapper}>
+          {isFetching ? (
+            <Loader />
+          ) : !isResultEmpty ? (
+            this.renderItems()
+          ) : (
+            'We have nothing for this request...'
+          )}
```

Comment:
Not really widely used practice to use not `!` in cases when we can avoid using it: Let's do it this way:
```suggestion
          {isFetching ? (
            <Loader />
          ) : isResultEmpty ? (
          'We have nothing for this request...'
          ) : (
           this.renderItems()
          )}
```

Another option to avoid `!` is to replace `const isEmpty = ...` with `const hasData = ...`

The choice is all yours!